tests/integration/abi_roundtrip_test.py

```diff
@@ -142,12 +154,21 @@ def roundtrip_setup(abi_type):
 
 
 def test_abi_types_comprehensive():
-    top_level_names = {
-        tli.split("[")[0] if tli.startswith("pyteal") else tli.split("'")[1]
-        for tli in (
-            str(x) for x in (at[0] if isinstance(at, tuple) else at for at in ABI_TYPES)
+    top_level_names = set()
+    for at in ABI_TYPES:
+        at = at[0] if isinstance(at, tuple) else at
+        tli: str
+
+        tli = (
+            str(abi.NamedTuple)
+            if isclass(at) and issubclass(at, abi.NamedTuple)
+            else str(at)
         )
-    }
+        tli = tli.split("[")[0] if tli.startswith("pyteal") else tli.split("'")[1]
+
+        top_level_names.add(tli)
+
+    print(top_level_names)
```

Comment:
```suggestion
    print(f"{top_level_names=}")
```
That suggestion assumes you want to keep the printout. But you might also consider removing it altogether.

Reply:
oh boy, I forgot to delete this line in debugging. No worries on changing the test behavior.